parlai/zoo/dodecadialogue/build.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Pretrained models from The Dialogue Dodecathlon.
+
+https://arxiv.org/abs/1911.03768
+"""
+from parlai.core.build_data import built, download_models, get_model_dir
+import os
+import os.path
+
+
+def download(datapath):
+    model_name = 'dodecadialogue'
+    mdir = os.path.join(get_model_dir(datapath), model_name)
+    version = 'v1.0'
+    if not built(mdir, version):
+        opt = {'datapath': datapath}
+        fnames = ['dodecadialogue.tgz']
```

Comment:
hmm, how big is this file?

Also, can we use the checkpointing infra that our cornell friends wrote (idk, maybe that's only for datasets...)

Reply:
I might be tempted to break this up soon, in #2537 I noticed that downloading these large group files slows down tests, since we might split one test class across multiple hosts, so they're downloading more data than they need.

Reply:
it's 12.5 gb. Would you prefer that I made each a separate .tgz file?

The `download_models` function has a bit more nuance than what is provided by `DownloadableFile` at the moment so i'll keep this as is for now.